README.md

```diff
@@ -36,7 +36,12 @@ var hermes = require('runnable-hermes').hermesSingletonFactory({
 
 var jobCallback = function (data, done) {
   // perform operation w/ data
-  done(); // remove job from RabbitMQ queue
+  done(); // remove job from RabbitMQ queue (ack)
+
+  // optionally
+    // reject = true - negative acknowledge job completion
+    // requeue = true - requeue job and resubmit to worker
+  done(/* reject, requeue */);
```

Comment:
Positional arguments can get annoying overtime. 
They always require comments and looking at docs over-and-over.
Consider making this an options object or string (better for future support too like for nack if we ever use it).